examples/bigcommerce/README.md

```diff
@@ -17,9 +17,17 @@ This example includes a home page for listing products by category and a product
 
 ## Deploy your own
 
-Deploy your own with [Vercel](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fmakeswift%2Fmakeswift%2Ftree%2Fmain%2Fexamples%2Fbigcommerce&env=MAKESWIFT_SITE_API_KEY,BIGCOMMERCE_STORE_NAME,BIGCOMMERCE_STORE_HASH,BIGCOMMERCE_ACCESS_TOKEN&envDescription=Check%20step%203%20of%20the%20example%20README.md%20for%20details%20on%20where%20to%20find%20these%20values.&envLink=https%3A%2F%2Fgithub.com%2Fmakeswift%2Fmakeswift%2Ftree%2Fmain%2Fexamples%2Fbigcommerce%23using-this-repo&project-name=makeswift-bigcommerce-example&repo-name=makeswift-bigcommerce-example) or preview live with [StackBlitz](https://stackblitz.com/github/makeswift/makeswift/tree/main/examples/bigcommerce)
+Deploy your own with [Vercel](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fmakeswift%2Fmakeswift%2Ftree%2Fmain%2Fexamples%2Fbigcommerce&env=MAKESWIFT_SITE_API_KEY,BIGCOMMERCE_STORE_NAME,BIGCOMMERCE_STORE_HASH,BIGCOMMERCE_ACCESS_TOKEN&envDescription=Check%20step%203%20of%20the%20example%20README.md%20for%20details%20on%20where%20to%20find%20these%20values.&envLink=https%3A%2F%2Fgithub.com%2Fmakeswift%2Fmakeswift%2Ftree%2Fmain%2Fexamples%2Fbigcommerce%23using-this-repo&project-name=makeswift-bigcommerce-example&repository-name=makeswift-bigcommerce-example) or preview live with [StackBlitz](https://stackblitz.com/github/makeswift/makeswift/tree/main/examples/bigcommerce)
```

Comment:
evidently the widget for creating your button spits out `repo-name` which doesn't work. Just figured out the problem by comparing with the thirdweb example. 

Thanks @apledger

Reply:
We should report this to Vercel!